core/controllers/feedback.py

```diff
@@ -58,7 +60,11 @@ class ThreadHandler(base.BaseHandler):
 
     def get(self, exploration_id, thread_id):
         self.values.update({
-            'messages': feedback_services.get_messages(thread_id)})
+            'messages': feedback_services.get_messages(
+                exploration_id, thread_id)})
+        self.values.update(
```

Comment:
Nit: Following the format of the previous clause, put the '{' at the end of this line